javalanglib/src/main/scala/java/lang/Long.scala

```diff
@@ -232,4 +233,22 @@ object Long {
     else if (mp != 0) Integer.toOctalString(mp) + padOctal10(lp)
     else Integer.toOctalString(lp)
   }
+
+  /**
+   * Holds the maximal number of digits for each radix, so that a number of that radix
+   * can be converted to an Integer
+   */
+  private def maxExpForRadix(radix: Int): Int = {
+    @inline
+    @tailrec
+    def loop(exp: Int, m: Long): Int = {
+      val l: Long = m * radix
+      if (l <= Int.MaxValue) loop(exp+ 1, l)
+      else exp
+    }
+
+    if (radix < 2) -1
+    else if (radix == 2) 30
+    else loop(0, 1)
+  }
```

Comment:
Yes, could do as hard-coded. This was sort-of listed from the BigNumerics where on a code review comment was to calc the array. I can see value in both, but now the code is there I'm not sure it is best to completely delete as it begs "where do the number come from" (btw also this is where I got the names from, but the method is actually slightly different.)

So what is best - to delete the code. comment it, leave it but hard code the values?